.travis.yml

```diff
@@ -13,6 +13,8 @@ python:
 
 before_install:
   - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
```

Comment:
If this is for matplotlib, you can do without starting X-server:
```yaml
# use matplotlib backend that does not need DISPLAY
- export MPLBACKEND=agg
```

Reply:
@pavoljuhas Thanks! It was for matplotlib and I agree suppressing x-server might be a better idea. I've included this change.